Sources/RemoteMessagingTestsUtils/MockRemoteMessagePercentileStore.swift

```diff
@@ -19,12 +19,17 @@
 import Foundation
 import RemoteMessaging
 
-class MockRemoteMessagePercentileStore: RemoteMessagingPercentileStoring {
+public class MockRemoteMessagePercentileStore: RemoteMessagingPercentileStoring {
```

Comment:
Mocks moved to a dedicated mocks target (RemoteMessagingTestsUtils in this case) need to be made public so that the module could be imported without `@testable`